package-lock.json

```diff
@@ -0,0 +1,2271 @@
+{
```

Comment:
I don't think there is a need for a `package-lock.json` at the project root. 

Rather, there should be one in each of those folders:
*  `client/`
* `back-end/`